luigi/worker.py

```diff
@@ -419,6 +441,12 @@ class worker(Config):
     force_multiprocessing = BoolParameter(default=False,
                                           description='If true, use multiprocessing also when '
                                           'running with 1 worker')
+    task_process_context = Parameter(default=None,
```

Comment:
Did you consider making this a "list" by making a comma-separate name of classnames?

But I guess users can make their single context-manager itself have multiple ones.

Reply:
Yes, exactly, that would be the next step I could think of; about to add in a separate PR.

Reply:
This will throw a warning because `None` is not allowed in string parameter:
```python-traceback
UserWarning: Parameter "task_process_context" with value "None" is not of type string.
```

Reply:
> This will throw a warning because `None` is not allowed in string parameter:
> 
> ```
> UserWarning: Parameter "task_process_context" with value "None" is not of type string.
> ```

I wonder why can't it be defaulted to `""` instead, like in
```
task_process_context = Parameter(default="",
```
@ulzha any thoughts?

Reply:
Alternatively, you can leave the None default and make it an `OptionalParameter`